### PR TITLE
initial attempt

### PR DESCRIPTION
**Pull request by vixie**

### CODE REVIEW

**Comment by vixie** [comment]
https://github.com/vixie/cron/issues/9

File: cron.8 [hunk 1]
```diff
@@ -22,9 +22,11 @@
 cron \- daemon to execute scheduled commands (Vixie Cron)
 .SH SYNOPSIS
 .B cron
-.RB [ \-l
-.IR load_avg ]
+.RB [ \-m
```
Comment (millert) [inline cron.8:25]: FreeBSD cron uses -m to set the default MAILTO, perhaps -M could be used here instead?

File: do_command.c [hunk 2]
```diff
@@ -405,14 +405,43 @@ child_process(entry *e, user *u) {
...
-
-				if (strlens(MAILFMT, MAILARG, NULL) + 1
-				    >= sizeof mailcmd) {
-					fprintf(stderr, "mailcmd too long\n");
+				char mailcmd[MAX_COMMAND] = "";
+				const char *msg = NULL;
+
+				if (Mailer != NULL) {
+					if (strstrcount(Mailer, "%s") == 1) {
+						if (strlens(Mailer, mailto) + 1
```
Comment (millert) [inline do_command.c:413]: I think you need a NULL terminator for strlens() here.

**Review by millert — CHANGES_REQUESTED** [review]

File: do_command.c (hunk 2 above)
Comment (vixie) [reply do_command.c]: check it now please? i hope to have resolved your concerns.

**Pull request merged**